pages/erd.vue

```diff
@@ -1,18 +1,44 @@
 <template>
-  <div id="erd">
-    <VideoPlayer src="video/erd_video.mp4"></VideoPlayer>
-  </div>
+  <v-container>
+    <v-row>
```

Comment:
why would you add a grid layout to create row for the button? would it work with a <div> wrapper?

Reply:
The page will have more components soon. then the grid layout will be used more